portal-api/api-tests/v1/companies/companies.api-test.js

```diff
@@ -0,0 +1,99 @@
+const { COMPANY_REGISTRATION_NUMBER } = require('@ukef/dtfs2-common');
+const app = require('../../../src/createApp');
+const { MAKER, READ_ONLY, ADMIN } = require('../../../src/v1/roles/roles');
+const testUserCache = require('../../api-test-users');
+const { withClientAuthenticationTests } = require('../../common-tests/client-authentication-tests');
+const { withRoleAuthorisationTests } = require('../../common-tests/role-authorisation-tests');
+const { POSTCODE } = require('../../fixtures/postcode');
+const { companies } = require('../../../src/external-api/api');
+const { as, get } = require('../../api')(app);
+
+jest.unmock('../../../src/external-api/api');
```

Comment:
ooh this is interesting, do we want to do this (it feels this is making an actual call to `${EXTERNAL_API_URL}/companies/${registrationNumber}` which feels slightly scary and not what we want? :)